### PR TITLE
Update pulp deps

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/pulpcore/python-dotenv/python-dotenv.spec [hunk 1]
```diff
@@ -30,11 +34,6 @@ Summary:        %{summary}
 # Remove bundled egg-info
 rm -rf %{pypi_name}.egg-info
 
-# setup.py loads the contents of README.md and CHANGELOG.md into long_description
```
Comment (ekohl) [inline packages/pulpcore/python-dotenv/python-dotenv.spec:33]: Is this fixed?

Comment (evgeni) [reply packages/pulpcore/python-dotenv/python-dotenv.spec:33]: my local scratch builds worked, so I'd say yes.

File: packages/pulpcore/python-itypes/python-itypes.spec [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
 Name:           python-%{pypi_name}
-Version:        1.1.0
-Release:        2%{?dist}
+Version:        1.2.0
+Release:        1%{?dist}
 Summary:        Simple immutable types for python
 
 License:        BSD
-URL:            http://github.com/tomchristie/itypes
+URL:            http://github.com/PavanTatikonda/itypes
```
Comment (ekohl) [inline packages/pulpcore/python-itypes/python-itypes.spec:10]: This repo is archived. Looks like it was a dependency of `coreapi` which is now also archived. Are the pulpcore devs aware of this?

Comment (evgeni) [reply packages/pulpcore/python-itypes/python-itypes.spec:10]: I pinged them on IRC, yes.

Comment (evgeni) [reply packages/pulpcore/python-itypes/python-itypes.spec:10]: and opened https://pulp.plan.io/issues/6583

**Pull request merged**